<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="126" started-at="2020-01-26T18:13:56Z" finished-at="2020-01-26T18:13:56Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="126" started-at="2020-01-26T18:13:56Z" finished-at="2020-01-26T18:13:56Z">
      <class name="org.testng.DataProvider">
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String)[pri:0, instance:org.testng.DataProvider@326de728]" name="test" duration-ms="0" started-at="2020-01-26T13:13:56Z" finished-at="2020-01-26T13:13:56Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'username' is required by @Test on method test but has not been marked @Optional or defined
in C:\Users\ELCOT\AppData\Local\Temp\testng-eclipse--150937887\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'username' is required by @Test on method test but has not been marked @Optional or defined
in C:\Users\ELCOT\AppData\Local\Temp\testng-eclipse--150937887\testng-customsuite.xml
	at org.testng.internal.Parameters.createParameters(Parameters.java:155)
	at org.testng.internal.Parameters.createParameters(Parameters.java:355)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:447)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1389)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1081)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1186)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:207)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:178)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="test1(java.lang.String)[pri:0, instance:org.testng.DataProvider@326de728]" name="test1" duration-ms="15" started-at="2020-01-26T13:13:56Z" finished-at="2020-01-26T13:13:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[sridhar]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
      </class> <!-- org.testng.DataProvider -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
